src/runner/index.js

```diff
@@ -87,14 +93,23 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    async run ({ skipJsErrors, quarantineMode } = {}) {
+    run ({ skipJsErrors, quarantineMode } = {}) {
         this.opts.skipJsErrors   = !!skipJsErrors;
         this.opts.quarantineMode = !!quarantineMode;
 
-        var { reporterPlugin, browserSet, tests } = await this.bootstrapper.createRunnableConfiguration();
+        var getRunTaskPromise = this.bootstrapper.createRunnableConfiguration()
+            .then(({ reporterPlugin, browserSet, tests }) => {
+                this.emit('done-bootstrapping');
+
+                // NOTE: If then-able is returned from onResolved/onRejected handler, it will be resolved
+                // So in order to get promise from `then` block, it needs to be wrapped in another object
```

Comment:
```
                // NOTE: If a thenable is returned from the onResolved/onRejected handler, it will be resolved.
                // So, in order to get a promise from a `then` block, we need to wrap it in another object.
```